src/fty-alert-engine-log.cfg

```diff
@@ -20,8 +20,8 @@ log4cplus.logger.alerts-engine-audit=DEBUG, alerts-engine-audit-log, syslog, con
 
```

Comment:
Do we need to add append property to true or is it by default active.
We should not erase file at restart of the service

Reply:
Without option, it is the default behavior (append)

Reply:
done active by default